IMPLEMENTATION_GUIDE.md

```diff
@@ -593,6 +595,58 @@ Returns:
   b'{"ApiVersion":"2.0.0","FrameworkVersion":"v2.0.4"}'
   ```
 
+#### `POST /service/transcribe/describe_language_model`
```

Comment:
I made it a post because it requires an input parameter for ModelName. Get_vocab and get_terminology also are post, so I think we're still being consistent with that pattern.